components/display/st7789v.rst

```diff
@@ -57,29 +57,26 @@ to an ESP module.
 
     For more information about the font options see: :ref:`display-fonts`.
 
-.. warning::
 
-    When using the TTGO T-Display module, the GPIO pin numbers above *cannot be changed* as they are
-    hardwired within the module/PCB.
 
 Configuration variables:
 ************************
 
 - **model** (**Required**, string): The display model to use. One of the following options:
 
-  - ``TTGO TDisplay 135x240``
+  - ``TTGO TDisplay 135x240`` Automatically specifies required pins.
   - ``Adafruit Funhouse 240x240``
   - ``Adafruit RR 280x240`` (round-rectangular display -- some pixels are "deleted" from corners to form rounded shape)
   - ``Adafruit S2 TFT FEATHER 240X135`` (requires ``power_supply`` be specified, see below)
+  - ``LILYGO T-Embed 170X320`` requires ``power_supply`` be specified, but automatically specifies ``dc_pin``, ``cs_pin``,
+    ``backlight_pin`` and ``reset_pin``.
```

Comment:
It might be a bit cleaner to add a `note` below this section to call out which pins are defined/handled by the preset. 

For example, `*` = add pins defined, `**` = all except `power_supply`, etc.
...or, as an alternative, a sub-bullet below each model with this information.
...or, as another alternative, a link to a table below which maps all this out.

I think it's more readable/cleaner than trying to in-line it.